pages/login.tsx

```diff
@@ -12,5 +12,5 @@ export default function LoginPage() {
     dispatch(clearExpectedError());
   }, [dispatch]);
 
-  return <Login returnToPath={"" + router.query.returnTo} />;
+  return <Login returnToPath={"" + (router.query.returnTo || "/")} />;
```

Comment:
This eliminates the requirement that `returnTo` be provided on the query string. Without the fallback `returnTo` was stringified to `"undefined"` by concatenating it with `""`.